sops/adding_nodes.md

```diff
@@ -0,0 +1,5 @@
+# Adding Compute/Worker nodes
+To add new worker/compute nodes to the OCP cluster see the detailed steps at: 
+
+- https://access.redhat.com/solutions/4246261[How to add Openshift 4 RHCOS worker nodes in UPI]
```

Comment:
Is this syntax correct?
I am only aware of `[]()`

Reply:
I'm writing asciidoc in my head.. ill fix this. :laughing: